src/endpoint/s3/ops/s3_get_object.js

```diff
@@ -52,14 +52,19 @@ async function get_object(req, res) {
     if (md_params.get_from_cache) {
         params.get_from_cache = true;
     }
-    if (part_number) {
-        params.part_number = part_number;
-    }
```

Comment:
Hmm this means that we now also affect how all namespaces also process reads with part number.
I'm thinking on NS-S3 for example - so now instead of passing part number to the GET request, we only send it on HEAD and then send a GET with a range. Not sure what to think of it yet, but I wonder if this is what we desire. We could limit this only to NS-NB if we prefer. WDYT? 
FYI @nimrod-becker @jeniawhite

Reply:
I think that since start\end are only returned by namespace_nb we can set `params.part_number` if `req.query.partNumber` exists and there is no start\end on the `object_md`

Reply:
Restoring